src/encoder/plugins/OpusEncoderPlugin.cxx

```diff
@@ -52,30 +53,33 @@ class OpusEncoder final : public OggEncoder {
 
 	ogg_int64_t packetno = 0;
 
-	ogg_int64_t granulepos;
+	ogg_int64_t granulepos = 0;
```

Comment:
Why does this need to be initialized now? Was this a bug previously, or did your other code changes require it?

Reply:
>Was this a bug previously
probably. it's not initialised anywhere else. technically granulepos in stream can start from any value, so decoders probably didn't complain and it was unnoticed.

Reply:
You're right, so this change has nothing to do with this patch. Separate patch, please, so I can cherry-pick this into the stable branch v0.20.x

Reply:
@MaxKellermann https://github.com/MusicPlayerDaemon/MPD/pull/222